docker/jenkins/run_functional_tests.sh

```diff
@@ -1,45 +1,67 @@
-#!/bin/bash
-#
-# Starts Selenium Hub and NUMBER_OF_NODES (default 5) firefox nodes.
-# Waits until at least one node is ready and then runs smoke tests
-# only against a local bedrock instance started for this job.
-#
-set -xe
+#!/bin/bash -xe
+cp docker/dockerfiles/bedrock_functional_tests Dockerfile
+docker build -t bedrock_functional_tests:${GIT_COMMIT} --pull=true .
```

Comment:
I think https://github.com/mozilla/bedrock/pull/3760/files#diff-4db431ac309eff3c872644e2c0ee9b9bR10 is likely the source of the merge conflict, and if you include that change when you rebase it will hopefully make things a little more convenient when running locally.